ngs_mapper/runsample.py

```diff
@@ -440,25 +441,45 @@ def select_keys(d, keys):
                 logger.critical( "{0} failed to complete sucessfully. Please check the log file {1} for more details".format(cmd,bwalog) )
                 sys.exit(1)
 
-        # Tag Reads
-        cmd = 'tagreads {bamfile} -CN {CN}'
-        if cmd_args['config']:
-            cmd += ' -c {config}'
-        p = run_cmd( cmd.format(**cmd_args), stdout=lfile, stderr=subprocess.STDOUT )
+        tcmd = "samtools view -F 2048 -b {tmp_bam} > {bamfile}"
+        p = run_cmd( tcmd.format(**cmd_args), stdout=lfile, stderr=subprocess.STDOUT )
         r = p.wait()
         if r != 0:
             logger.critical( "{0} did not exit sucessfully".format(cmd.format(**cmd_args)) )
         rets.append( r )
 
-        # Variant Calling
-        cmd = 'base_caller {bamfile} {reference} {vcf} -minth {minth}'
+        # Tag Reads
+        cmd = 'tagreads {bamfile} -CN {CN}'
         if cmd_args['config']:
             cmd += ' -c {config}'
         p = run_cmd( cmd.format(**cmd_args), stdout=lfile, stderr=subprocess.STDOUT )
         r = p.wait()
         if r != 0:
             logger.critical( "{0} did not exit sucessfully".format(cmd.format(**cmd_args)) )
         rets.append( r )
+        # Variant Calling
+        from ngs_mapper.config import load_config
+        bc_cfg = load_config(cmd_args['config'])['base_caller']
```

Comment:
Only way to use this is to edit config file and use that config file right?

Maybe we can add a flag to run_sample.py as well?